lib/dl_connector_trino/dl_connector_trino/core/connector.py

```diff
@@ -8,6 +8,7 @@
 from dl_core.data_source_spec.sql import StandardSchemaSQLDataSourceSpec
 from dl_core.us_manager.storage_schemas.data_source_spec_base import SchemaSQLDataSourceSpecStorageSchema
 
+from dl_connector_trino.core import vendor_patches  # noqa: F401 # BI-6846
```

Comment:
should be moved to root of dl_connector_trino, also add comment about possible fix in trino repo